### ISSUE TITLE
Cambiar título: Crear/Modificar 

### ISSUE THREAD

**Issue by jgsogo** [Lingwars/blog#9]
El artículo "[Crear contenidos en el blog](https://github.com/Lingwars/blog/blob/master/content/Tutoriales/publish_content.md)" cuenta también cómo modificar un blog existente. ¿Habría que cambiarle el título?  ¿Te atreves @nimbusaeta?

(Como véis se pueden crear tareas, e incluso asignárselas a otra persona)

**Comment by nimbusaeta** [comment]
Quieres decir que también explica cómo modificar un _artículo_ existente, ¿no? Es que me lío xD ok, a ver si puedo cambiar el título

**Comment by jgsogo** [comment]
Yes! Es una modificación muy sencilla para empezar a probar cómo funciona esto de Git (aunque ya te estás haciendo un máster con un pull-request con los otros dos artículos)
